package.json

```diff
@@ -12,6 +12,7 @@
     "typescript": "^4.8.3"
   },
   "scripts": {
-    "bootstrap": "lerna bootstrap --hoist --force-local --ignore-prepublish"
+    "bootstrap": "lerna bootstrap --ci --hoist --force-local --ignore-prepublish",
+    "bootstrap-no-drivers-no-app": "lerna bootstrap --ignore reltab-* --ignore tad --ci --hoist --force-local --ignore-prepublish"
```

Comment:
What does the `--ignore tad` option do here?  According to the lerna docs it is a `<glob>` and based on the previous argument (`reltab-*`) I would guess it is matching against either the `packages/` directory or the `packages` property in `lerna.json`.  But in neither case can I see how `tad` matches anything if it doesn't have a wildcard.

Reply:
Somewhat unintuitively, this glob [matches against the package name in package.json](https://www.npmjs.com/package/@lerna/bootstrap#--ignore), not the directory. So this matches tad-app, which is named "tad" in packge.json.

I tried also ignoring "tadweb-app" and tadweb-server", but for some reason these didn't stick, and I couldn't figure out why.